README.md

```diff
@@ -80,7 +83,7 @@ public class IntroActivity extends AppIntro {
 }
 ```
 
-Finally, declare the activity in your Manifest like so:
+Finally, declare the activity in your Manifest like that:

```

Comment:
This was correct with the word so...you might also say "like this" because you're showing people what to code.